cherrypy/lib/encoding.py

```diff
@@ -321,9 +321,9 @@ def gzip(compress_level=5, mime_types=['text/html', 'text/plain'],
     values in the mime_types arg before calling this function.
 
     The provided list of mime-types must be of one of the following form:
-        * type/subtype
-        * type/*
-        * type/*+subtype
+        * `type/subtype`
```

Comment:
My limited tests indicated that it works better than the original version. And at the very least, it doesn't generate errors.